### PR TITLE
Adds github bot

### PR DESCRIPTION
**Pull request by gidsi**
Add the github directory check bot to the server.

### CODE REVIEW

File: deployment/playbooks/provisioning.yml [hunk 1]
```diff
@@ -3,15 +3,17 @@
...
     - { role: storage }
     - { role: base, tags: ['base'] }
+    - { role: motd, tags: ['motd'] }
+
+    # services
     - { role: mattermost, tags: ['mattermost'] }
     - { role: matterbridge, tags: ['matterbridge'] }
-    - { role: motd, tags: ['motd'] }
-    - { role: docker, tags: ['docker'] }
-    - { role: watchtower, tags: ['watchtower'] }
```
Comment (dbrgn) [inline deployment/playbooks/provisioning.yml]: Why remove docker and watchtower? This way we can't easily change watchtower configuration without having to redeploy any service that depends on watchtower.

Comment (gidsi) [reply deployment/playbooks/provisioning.yml]: You're right. Didn't thought about that.

**Comment by gidsi** [comment]
> Can you summarize what Github Bot does?

The bot checks the added or changed url in any pull request in the directory repo for reachability, correct headers and validity. It adds the outcome as a check onto the pull request and displays the outcome of it.

Right now its more like a MVP, it doesn't comment on the pull request and doesn't gives a hint what to fix and it still needs a database.

I would like to change that on the next version, but right now it already takes the need to do the checks all by hand.

**Comment by dbrgn** [comment]
Ah, *this* bot :slightly_smiling_face: Cool.
